hrms/payroll/doctype/payroll_entry/payroll_entry.js

```diff
@@ -136,9 +136,29 @@ frappe.ui.form.on('Payroll Entry', {
 			frm.add_custom_button(__("Create Salary Slips"), function() {
 				frm.trigger("create_salary_slips");
 			}).addClass("btn-primary");
+		} else if (
+			(!frm.doc.journal_entry_submitted
+			&& frm.doc.salary_slips_submitted)
+			|| (frm.doc.__onload && frm.doc.__onload.submitted_ss)
+		) {
+			frm.events.add_journal_entry_button(frm);
 		}
 	},
 
+	add_journal_entry_button: function(frm){
+		frm.add_custom_button(__("Create Journal Entry"), function () {
```

Comment:
```suggestion
		frm.add_custom_button(__("Make Accrual Journal Entry"), function () {
```
Can we call this button Make Accrual Journal Entry to be inline with "Make Bank Entry"?
Accordingly change the function names, labels and messages on client-side
Also, make it primary